<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictListManagerDialog</class>
 <widget class="QDialog" name="DictListManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dictionary Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListView" name="listView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="infoBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>DictListManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upBtn</sender>
   <signal>clicked()</signal>
   <receiver>DictListManagerDialog</receiver>
   <slot>onUpBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downBtn</sender>
   <signal>clicked()</signal>
   <receiver>DictListManagerDialog</receiver>
   <slot>onDownBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addBtn</sender>
   <signal>clicked()</signal>
   <receiver>DictListManagerDialog</receiver>
   <slot>onAddBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeBtn</sender>
   <signal>clicked()</signal>
   <receiver>DictListManagerDialog</receiver>
   <slot>onRemoveBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infoBtn</sender>
   <signal>clicked()</signal>
   <receiver>DictListManagerDialog</receiver>
   <slot>onInfoBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onUpBtnClicked()</slot>
  <slot>onDownBtnClicked()</slot>
  <slot>onAddBtnClicked()</slot>
  <slot>onRemoveBtnClicked()</slot>
  <slot>onInfoBtnClicked()</slot>
  <slot>onListSelectionChanged()</slot>
 </slots>
</ui>
